### PR TITLE
CI-4054 adding `AEM as a Cloud Service` integration docs 

### PR DESCRIPTION
**Pull request by francoisledroff**
* extracted the `on-premise` AEM installation doc
* created an `AEM as a Cloud Service` similar doc

### CODE REVIEW

File: aem/aem_skyline_install.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+
+## Integrate with AEM as a Cloud Service
+
+This documentation is specific to `AEM as a Cloud Service` set up, to integrate with `AEM on premise` 
+please refer to the other associated [documentation](aem_on_premise_install.md).
+
+### Pre-release provisioning
+
+If you are an existing `AEM as a Cloud Service` customer, please contact your Adobe representative providing your AEM authors public urls.
```
Comment (rperelma) [inline aem/aem_skyline_install.md:9]: author's

File: aem/aem_skyline_install.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+For that, log-in to [your cloud manager](my.cloudmanager.adobe.com/) space, look up your authors environments
```
Comment (rperelma) [inline aem/aem_skyline_install.md:11]: author's

File: aem/aem_skyline_install.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+  ![Cloud Manager Environments](../img/cloud_manager_environments.png "Cloud Manager Environments")
+
+Select from this list the environments public url you want Adobe I/O to expose as `Events Providers` .
```
Comment (rperelma) [inline aem/aem_skyline_install.md:15]: environment's

File: aem/aem_skyline_install.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+If you are an existing `AEM as a Cloud Service` customer, please contact your Adobe representative providing your AEM authors public urls.
+
+For that, log-in to [your cloud manager](my.cloudmanager.adobe.com/) space, look up your authors environments
+
+  ![Cloud Manager Environments](../img/cloud_manager_environments.png "Cloud Manager Environments")
+
+Select from this list the environments public url you want Adobe I/O to expose as `Events Providers` . 
+
+As soon as the Adobe I/O team is done provisioning it, your new `AEM as a Cloud Service` `Events Providers` should appear in the Adobe I/O console.
```
Comment (rperelma) [inline aem/aem_skyline_install.md:17]: This part is not clear to the reader, they don't know who the Adobe I/O team is, or what they need to ask to make this happen. Not sure if @kgera has a suggestion for how to improve this.

File: aem/aem_skyline_install.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+### Ping Events 
+
+Now you can start verifying that your `AEM as a Cloud Service` environment is emitting events to Adobe I/O.
+To do that register a new [webhook](../intro/webhook_docs_intro.md) or start pulling events from this new source using the [journaling API](../intro/journaling_api.md).
+
+Make sure you register to receive `OSGI Ping Event`
+
+   ![Select OSGI Ping Event](../img/select_osgi_ping_event.png "Select OSGI Ping Event")
+ 
+If your webhook regitration is active, every minutes or so,
```
Comment (rperelma) [inline aem/aem_skyline_install.md:88]: minute (singular)

File: aem/aem_skyline_install.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
+
+## Integrate with AEM as a Cloud Service
+
+This documentation is specific to `AEM as a Cloud Service` set up, to integrate with `AEM on premise` 
+please refer to the other associated [documentation](aem_on_premise_install.md).
```
Comment (shikhartanwar) [inline aem/aem_skyline_install.md:5]: Shall we break this statement into two separate statements?
```suggestion
This documentation is specific to `AEM as a Cloud Service` set up.
To integrate with `AEM on premise`, please refer to the other associated [documentation](aem_on_premise_install.md).
```

File: aem/aem_skyline_install.md (hunk 1 above)
Comment (shikhartanwar) [inline aem/aem_skyline_install.md:9]:
```suggestion
If you are an existing `AEM as a Cloud Service` customer, please contact your Adobe representative for providing your AEM author's public urls.
```

File: aem/aem_skyline_install.md (hunk 4 above)
Comment (shikhartanwar) [reply aem/aem_skyline_install.md:17]: I agree. Maybe we should add an external facing DL with both our team members and some of Skyline team members and quote that DL here.

File: aem/aem_skyline_install.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+For that, log-in to [your cloud manager](my.cloudmanager.adobe.com/) space, look up your authors environments
+
+  ![Cloud Manager Environments](../img/cloud_manager_environments.png "Cloud Manager Environments")
+
+Select from this list the environments public url you want Adobe I/O to expose as `Events Providers` . 
+
+As soon as the Adobe I/O team is done provisioning it, your new `AEM as a Cloud Service` `Events Providers` should appear in the Adobe I/O console.
+ 
+  ![An Adobe I/O Console `AEM as a Cloud Service` Events Provider](../img/add_skyline_event_provider.png "An Adobe I/O Console `AEM as a Cloud Service` Events Provider")
```
Comment (shikhartanwar) [inline aem/aem_skyline_install.md:19]: Also, I am not sure if this is easily understood by the outside world. Should we add more screenshots on how to access this provider from console?

File: aem/aem_skyline_install.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+Once these 2 setups are done, you should have:
+
+* created a public/private certificate key and an associated keystore
+* added the keystore into the AEM `eventproxy-service` user&rsquo;s keystores vault.
+* created an Adobe I/O Console project and workspace using the public certificate
+
+You are now ready to finalize the Adobe IMS configuration needed by Adobe I/O Events:
+
+1. Open the Cloud Services console, or select the **Security** icon, and then select **Adobe IMS Configurations**.
```
Comment (shikhartanwar) [inline aem/aem_skyline_install.md:50]: Cloud Services console?

File: aem/aem_skyline_install.md (hunk 5 above)
Comment (shikhartanwar) [inline aem/aem_skyline_install.md:88]: Please also fix the typo -
```suggestion
If your webhook registration is active, every minute or so, 
```

File: aem/aem_skyline_install.md (hunk 1 above)
Comment (francoisledroff) [reply aem/aem_skyline_install.md]: done

File: aem/aem_skyline_install.md (hunk 2 above)
Comment (francoisledroff) [reply aem/aem_skyline_install.md]: done

File: aem/aem_skyline_install.md (hunk 3 above)
Comment (francoisledroff) [reply aem/aem_skyline_install.md]: rephrased

File: aem/aem_skyline_install.md (hunk 4 above)
Comment (francoisledroff) [reply aem/aem_skyline_install.md]: I rephrased this section and remove the reference to the Adobe I/O team.

File: aem/aem_skyline_install.md (hunk 5 above)
Comment (francoisledroff) [reply aem/aem_skyline_install.md]: done

File: aem/aem_skyline_install.md (hunk 6 above)
Comment (francoisledroff) [reply aem/aem_skyline_install.md]: done

File: aem/aem_skyline_install.md (hunk 1 above)
Comment (francoisledroff) [reply aem/aem_skyline_install.md]: rephrased

File: aem/aem_skyline_install.md (hunk 7 above)
Comment (francoisledroff) [reply aem/aem_skyline_install.md:22]: I added a link to a related doc

File: aem/aem_skyline_install.md (hunk 8 above)
Comment (francoisledroff) [reply aem/aem_skyline_install.md]: good catch it's the `Tools` section not the `Cloud Services`

File: aem/aem_skyline_install.md (hunk 5 above)
Comment (francoisledroff) [reply aem/aem_skyline_install.md]: done

File: aem/aem_skyline_install.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+## Integrate with AEM as a Cloud Service
+
+This documentation is specific to `AEM as a Cloud Service` set up.
+
+To integrate with `AEM on premise`, please refer to the other associated [documentation](aem_on_premise_install.md).
+
+### Pre-release provisioning
+
+If you are an existing `AEM as a Cloud Service` customer, please contact your Adobe representative
+and provide him with the list of AEM author's public urls you want Adobe I/O to expose as `Events Providers`.
```
Comment (rperelma) [inline aem/aem_skyline_install.md:11]: ... and provide them...   (not: him)

**Pull request merged**